stable/redis-ha/templates/redis-server-pdb.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name: {{ template "redis-ha.fullname" . }}-server-pdb
```

Comment:
Why not add the default labels here as well?

Reply:
Will do